packages/lesswrong/components/posts/PostsCommentsThread.jsx

```diff
@@ -1,31 +1,36 @@
-import React from 'react';
+import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
-import { withList, Components, registerComponent, Utils} from 'meteor/vulcan:core';
+import { withList, Components, registerComponent } from 'meteor/vulcan:core';
 import { Comments } from 'meteor/example-forum';
+import { unflattenComments } from "../../lib/modules/utils/unflatten";
 
+class PostsCommentsThread extends PureComponent {
+  constructor(props) {
```

Comment:
Why do you need the constructor here at all? It doesn't seem to do anything, and there is no state.